builtin/mainmenu/content/pkgmgr.lua

```diff
@@ -840,7 +840,7 @@ function pkgmgr.get_contentdb_id(content)
 		return content.author:lower() .. "/" .. content.name
 	end
 
-	-- Until Minetest 5.8.0, Minetest Game was bundled with Minetest.
+	-- Until Luanti 5.8.0, Minetest Game was bundled with Luanti.
```

Comment:
```suggestion
	-- Until version 5.8.0, Minetest Game was bundled with Luanti.
```

As it wasn't called Luanti back then